content/template-tag-codemod.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Try out the Template Tag Codemod
+authors:
+  - edward-faulkner
+date: 2025-02-21T00:00:00.000Z
+tags:
+  - polaris
+  - '2025'
+  - template-tag
+  - embroider
+---
+
+One of the pillars of Ember's upcoming Polaris edition is [Template Tag](https://guides.emberjs.com/release/components/template-tag-format/). Many teams are already writing their components this way and enjoying the improved developer experience. The Template Tag RFC is currently in the "Released" status, and will reach "Recommended" [once we're satisfied that we have sufficient documentation and polish](https://github.com/emberjs/rfcs/pull/1059).
+
+To make it easier for all teams to adopt Template Tag, we've created `@embroider/template-tag-codemod`. The goal of the codemod is fully-reliable conversion to the new format.
+
+_Why is this codemod under the `@embroider` namespace?_
```

Comment:
I think it would be okay to bump this section down below the list of actions to take